backend/inspirehep/submissions/marshmallow/conferences.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""JSON Schemas."""
+
+import pycountry
+from inspire_schemas.builders import ConferenceBuilder
+from inspire_utils.record import get_value
+from marshmallow import Schema, fields, post_load
+
+
+class Conference(Schema):
+    additional_info = fields.Raw()
+    addresses = fields.Raw()
+    acronyms = fields.Raw()
+    contacts = fields.Raw()
+    dates = fields.Raw()
+    description = fields.Raw()
+    field_of_interest = fields.Raw()
+    keywords = fields.Raw()
+    name = fields.Raw()
+    series_name = fields.Raw()
+    series_number = fields.Raw()
+    subtitle = fields.Raw()
+    websites = fields.Raw()
+
+    @post_load
+    def build_conference(self, data) -> dict:
+        """Process data from form and build a conference record.
+
+        Args:
+            data (ConferenceRecord): record to serialize
+
+        Return:
+            dict: a conference record
+
+        """
+        country_mapping = {country.name: country.alpha_2 for country in pycountry.countries}
```

Comment:
instead of creating a mapping, you can directly access country by `name` and get `code` property with `pycountry.countries.get(name='<country_name>').alpha_2`